src/index.js

```diff
@@ -18,7 +35,28 @@ function renderSidebar() {
     <ul class="email-list">${messageLIs}</ul>
   `;
   const container = document.querySelector('.email-list');
-  if (container != null) container.innerHTML += sidebarContents;
+  if (container != null) container.innerHTML = sidebarContents;
+}
+
+/* Called by addNavEventListeners(), this function invokes the renderSidebar
+function and toggles the "active" class for styling purposes */
+function renderMailbox(eventTarget) {
+  const mailboxName = eventTarget.textContent;
+  const activeMB = document.querySelector('.active');
+  activeMB.classList.remove('active');
+  eventTarget.classList.add('active');
+  renderSidebar(mailboxName);
+}
```

Comment:
This is now a third different form of a function called `renderSomething`. This one doesn't generate any HTML strings, which seems to me like the core of “rendering.” Can you come up with a more descriptive name for this? (It might also make the argument of an event or event target more natural.)